src/util/kaldi-table-inl.h

```diff
@@ -340,6 +441,13 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
     // or the user called FreeCurrent().. note,
     // if when called by user we are in this state,
     // it means the user called FreeCurrent().
+    kObjectLoaded, // Called NextScpLine() and found we already  yes         yes
```

Comment:
Can you please go through the files and make sure that wherever I made a
comment, you've addressed it?  You seem only to have addressed some of my
latest comments.

On Sat, Mar 26, 2016 at 12:24 AM, Xiaohui Zhang <notifications@github.com>
wrote:

> In src/util/kaldi-table-inl.h
> <https://github.com/kaldi-asr/kaldi/pull/611#discussion_r57509107>:
>
> > @@ -340,6 +441,13 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
> >      // or the user called FreeCurrent().. note,
> >      // if when called by user we are in this state,
> >      // it means the user called FreeCurrent().
> > +    kObjectLoaded, // Called NextScpLine() and found we already  yes         yes
>
> ok that where the confusion happened. thanks for the explanation. fixed.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/611/files/d2c8803477273b8c96d2aacebdba9f93ea780781#r57509107>
>